dspace-api/src/main/java/org/dspace/content/CollectionServiceImpl.java

```diff
@@ -1068,4 +1070,44 @@ public int countCollectionsWithSubmit(String q, Context context, Community commu
     public int countArchivedItems(Context context, Collection collection) {
         return itemCounter.getCount(context, collection);
     }
+
+    private void cleanupLegacyWorkflowGroups(Context context,
+            Collection collection, List<CollectionRole> collectionRoles)
+            throws SQLException, AuthorizeException, IOException {
+
+        for (CollectionRole collectionRole : collectionRoles) {
+            Group g = collectionRole.getGroup();
+            if (g != null) {
+                String roleId = collectionRole.getRoleId();
+                int stepId = -1;
+                switch (roleId) {
+                    case "reviewer":
```

Comment:
Please don't hardcode the values here. Instead, you should be using `CollectionRoleService.LEGACY_WORKFLOW_STEP1_NAME`.

Same with the other hardcoded values below, there's variables that can replace each if you use `CollectionRoleService` directly